test/network.bats

```diff
@@ -118,3 +118,17 @@ load helpers
 
 	stop_crio
 }
+
+@test "Clean up network if pod sandbox fails" {
+	rm -f /var/lib/cni/networks/crionet_test_args/*
+	start_crio "" "" "" "prepare_plugin_test_args_network_conf"
+	# kill conmon to ensure pod fails to start after networking is set up
+	kill -9 $(ps -C conmon -o pid=)
```

Comment:
There should be no conmon running without starting a pod or container.

Reply:
@mrunalp ah, ok.  what I'm trying to do is cause a failure of sandbox setup *after* network has been correctly set up.  When testing this locally I had an old conmon installed and that did the trick, any suggestions here?  The only operations after network are:
```
	err = g.SaveToFile(filepath.Join(podContainer.Dir, "config.json"), saveOptions)
	if err != nil {
		return nil, fmt.Errorf("failed to save template configuration for pod sandbox %s(%s): %v", sb.Name(), id, err)
	}
	if err = g.SaveToFile(filepath.Join(podContainer.RunDir, "config.json"), saveOptions); err != nil {
		return nil, fmt.Errorf("failed to write runtime configuration for pod sandbox %s(%s): %v", sb.Name(), id, err)
	}

	if err = s.runContainer(container, sb.CgroupParent()); err != nil {
		return nil, err
	}

	s.addInfraContainer(container)

	s.ContainerStateToDisk(container)
```

Reply:
perhaps I could have a wrapper script for the network plugin that removes podContainer.Dir or something...

Reply:
@dcbw Yeah, something like that should work.